### PR TITLE
VTCCA-4416 bugs fixed

### PR DESCRIPTION
**Pull request by reddy-yattapu**

### CODE REVIEW

**Comment by darrell-plant** [comment]
LGTDz 👍

File: app/views/helpers/dateFields.scala.html [hunk 1]
```diff
@@ -45,7 +45,7 @@
         item =>
             InputItem(
                 id = s"${id.getOrElse(fieldName).replace(".", "-")}-$item",
-                classes = s"govuk-input--width-${if(item == "year") "4" else "2"} ${if(form.hasErrors) "govuk-input--error"}",
+                classes = s"govuk-input--width-${if(item == "year") "4" else "2"} ${if(!dateErrors.isEmpty) "govuk-input--error"}",
```
Comment (habeeb01) [inline app/views/helpers/dateFields.scala.html:48]: can use `nonEmpty` ??? - just a suggestion

**Comment by habeeb01** [comment]
LGTM 👍

File: app/views/helpers/dateFields.scala.html (hunk 1 above)
Comment (reddy-yattapu) [reply app/views/helpers/dateFields.scala.html]: changed to if(dateErrors. nonEmpty)

**Pull request merged**